nginx/ngx_js_fetch.c

```diff
@@ -277,6 +277,16 @@ static njs_int_t ngx_fetch_flag_set(njs_vm_t *vm, const ngx_js_entry_t *entries,
 static njs_int_t ngx_js_fetch_init(njs_vm_t *vm);
 
 
+njs_inline void
```

Comment:
Let's make it an ordinary static function and move somewhere down. 
It is up to compiler to inline it.